TECHNICAL_DOCUMENTATION.md

```
@@ -332,25 +333,22 @@ User: /verify-link github_username
       └─> Audit event: identity_verified
 ```
 
-**Issue Request Flow:**
+**Profile & Metrics:**
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)